.github/workflows/go.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         fi
 
     - name: Test
-      run: go test -v -race ./... -timeout 15m
+      run: go test -v -race ./... -timeout 30m
```

Comment:
This will increase the timeout in normal CI which is not failing (and hence the current 10m is fine). The one that is failing is the scheduled CI for which `go-scheduled.yml` is responsible.

If you see that file, you can see the line `go test -v -race -timeout=30m ./pkg/tests/end_to_end_tests/ -extended-test` is responsible for running all tests that need an increased timeout of 30m. This is the one that is failing. But, if you note, this test already has increased timeout, which mean something else if causing the hault.